ArduCopter/RC_Channel.cpp

```diff
@@ -48,6 +48,13 @@ bool RC_Channels_Copter::has_valid_input() const
     return true;
 }
 
+bool RC_Channels_Copter::arming_check_throttle() const {
```

Comment:
Let's put some comments somewhere.. nothing long and wordy but a quick description of what it's doing will be helpful for the next dev who treads over this path.

Reply:
done